src/components/victory-chart.jsx

```diff
@@ -425,10 +429,21 @@ export default class VictoryChart extends React.Component {
     return _formatData(childData);
   }
 
+  determineCategoryIndex(x, categories) {
```

Comment:
I think this should live in `victory-util/src/domain` or whatever makes sense, since the implementation is identical to that in `victory-bar`. Same with `padDomain`, `getCumulativeData`, etc. Basically any business logic that can be extracted from components and shared, should be.